### DISCUSSION TITLE
Used bandwidth in S3 storage

### DISCUSSION THREAD

**Discussion by renggap** [AzuraCast/AzuraCast#3925] [Q&A] [answered]
Hello, 

If I choose S3 storage for files, when it streamed to listener, which bandwidth to be used? My AzuraCast instance or S3 bandwidth?

**Comment by BusterNeece** [comment, marked as answer]
@renggap Your AzuraCast instance bandwidth is the one that will go up or down depending on how many listeners you have. Your S3 bandwidth will stay the same no matter how many listeners you have, as the connection between AzuraCast and S3 will only ever transfer the same number of files.
